pkgs/http/lib/src/base_response.dart

```diff
@@ -68,3 +70,68 @@ abstract class BaseResponse {
     }
   }
 }
+
+// "token" as defined in RFC 2616, 2.2
```

Comment:
[nit] Use `///` and make the comments on private members docs.

Reply:
Done